curriculum/challenges/english/10-coding-interview-prep/the-odin-project/html-boilerplate-question-c.md

```diff
@@ -0,0 +1,66 @@
+---
+id: 637f4e2f72c65bc8e73dfe22
+videoId: LGQuIIv2RVA
+title: HTML Boilerplate Question C
+challengeType: 11
+dashedName: html-boilerplate-question-c
+---
+
+# --description--
+
+The `<head>` element is where you put important meta-information about your webpages, and stuff required for your webpages to render correctly in the browser. Inside the `<head>`, you should not use any element that displays content on the webpage.
+
+## The Charset Meta Element
+You should always have the meta tag for the charset encoding of the webpage in the head element: `<meta charset="utf-8">`.
+
+Setting the encoding is very important because it ensures that the webpage will display special symbols and characters from different languages correctly in the browser.
+
+## Title Element
+Another element you should always include in the head of an HTML document is the title element:
+
+```html
+<title>My First Webpage</title>
+```
+
+The title element is used to give webpages a human-readable title which is displayed in your webpage’s browser tab.
+
+If you didn’t include a title element, the webpage’s title would default to its file name. In your case that would be `index.html`, which isn’t very meaningful for users; this would make it very difficult to find your webpage if the user has many browser tabs open.
```

Comment:
```suggestion
If you didn’t include a `title` element, the webpage’s title would default to its file name. In your case that would be `index.html`, which isn’t very meaningful for users; this would make it very difficult to find your webpage if the user has many browser tabs open.
```